### PR TITLE
Spaces overview refinement

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/6333
- Fixes https://github.com/owncloud/web/issues/6400

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/22616/55/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:253

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/22616/60/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:342

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 1]
```diff
@@ -6,20 +6,21 @@
...
-      <span v-text="$gettext('Access all project related files in one place.')" />
-      <a href="#" v-text="$gettext('Learn more about spaces.')" />
+    <span class="oc-display-block oc-mb-xl">
+      <span
+        v-text="$gettext('Store your project related files in Spaces for seamless collaboration.')"
+      />
+      <!-- <a href="#" v-text="$gettext('Learn more about spaces.')" /> -->
     </span>
-    <hr class="oc-mb-s" />
+    <hr class="oc-mb-l oc-border-b" />
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:23]: Feels a bit weird to give a bottom border to an `<hr>` 🤔  I'd prefer if you'd restyle the top border to be thicker instead of having a bottom and a top border applied at the same time.

By the way, accessibility wise, the `<hr>` tag will be announced via screen readers as horizontal divider. Was that intended? If not you'd need to give the block above a bottom border instead of adding the `<hr>`.

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 2]
```diff
@@ -6,20 +6,21 @@
...
       data-testid="spaces-list-create-space-btn"
       @click="showCreateSpaceModal"
     >
       <oc-icon name="add" />
       <translate>Create Space</translate>
     </oc-button>
-    <span class="oc-display-block oc-mb-l">
-      <span v-text="$gettext('Access all project related files in one place.')" />
-      <a href="#" v-text="$gettext('Learn more about spaces.')" />
+    <span class="oc-display-block oc-mb-xl">
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:17]: Why don't you use a `div`? Converting a `span` to `display: block` indicates to me that you want to have a div. 😅

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 3]
```diff
@@ -305,14 +313,11 @@ export default {
 .spaces-list {
   &-card {
     box-shadow: none !important;
+    border-radius: 3px;
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:316]: There is an `oc-rounded` css class from the design system. Could you try to use that instead of applying the 3px border radius here and in line 335ff?

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 4]
```diff
@@ -305,14 +313,11 @@ export default {
...
 
-    .space-context-btn {
-      z-index: 999;
-    }
-
-    .oc-card-media-top button {
-      top: 0;
-      right: 0;
+    .oc-card-body {
+      padding: 10px;
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:320]: possible to use the `m` sized spacing css variable? It's 8px. But IMO it still looks good.

Comment (kulmann) [reply packages/web-app-files/src/views/spaces/Projects.vue:320]: Basically: avoid using `px` values anywhere you can.

File: packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.js.snap [hunk 5]
```diff
@@ -1,39 +1,43 @@
...
+                  </button></li>
+              </ul>
+            </div>
+          </div>
         </div>
-      </li>
-    </ul>
   </div>
+  <p class="oc-text-left oc-mt-xs oc-mb-rm oc-text-truncate"><small></small></p></span>
+</div>
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.js.snap:38]: Having those additional closing tags here gives me the worry that the respective template has some missing closing tags somewhere, could you look into it?

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 6]
```diff
@@ -6,20 +6,21 @@
       ref="createNewSpaceButton"
       key="new-space-menu-btn-enabled"
       :aria-label="$gettext('Create a new space')"
-      variation="primary"
-      appearance="filled"
-      class="oc-mb-l"
+      variation="inverse"
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:9]: This should be fixed in the design system. The primary variation is supposed to look like you styled it manually now. Please revert the changes to the button here.

**Review by kulmann — CHANGES_REQUESTED** [review]
Overall really good improvement! Found some small issues. Also see attached screenshot: the `three dots` button in the spaces overview is not perfectly aligned vertically. And you need to add additional padding above the footer row of the card (below the hairline) because it looks off by the pixels that you have for the border radius at the bottom. ;-)

Screenshot 2022-02-10 at 09 46 54

File: packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.js.snap (hunk 5 above)
Comment (JanAckermann) [reply packages/web-app-files/tests/unit/views/spaces/__snapshots__/Projects.spec.js.snap:38]: don't see any tag fuck-up

File: packages/web-app-files/src/views/spaces/Projects.vue [hunk 7]
```diff
@@ -325,21 +323,39 @@ export default {
...
+  .oc-tag {
+    color: var(--oc-color-text-default);
+  }
+
   .space-image {
     width: 100%;
-    height: 200px;
+    height: 150px;
     object-fit: cover;
+    border-top-left-radius: 3px;
```
Comment (kulmann) [inline packages/web-app-files/src/views/spaces/Projects.vue:345]: You can use the css class `oc-rounded-top` here, which will set the top left and top right radius to 5px. FYI: We have helper classes for top, right, bottom, left and all around.

**Review by kulmann — APPROVED** [review]
🥳 🥳 🥳
